cgi-bin/admin.c

```diff
@@ -1225,6 +1236,8 @@ do_am_printer(http_t *http,		/* I - HTTP connection */
 
       cgiSetVariable("OP", "set-printer-options");
       do_set_options(http, 0);
+      if (ppd_name)
```

Comment:
Don't need the NULL check, and not sure we need to bother freeing it since the program just exits after the return...

Reply:
Hmm... isn't ppd_name NULL if we supply a PPD file via 'Or provide a PPD file'? And we reach these blocks in that case. So I guess it goes down to whether we want to free the memory or not here.

TBH, I've added the frees because I know our coverity always shouts at me that there are memory leaks in cgi scripts, so I wanted to prevent them in the PR I've created :D . I know they're not serious, since the program ends right after and then not-freed memory is handled by OS, but it seemed like a right thing to do (I can hear my college teacher: 'free your memory which you'd allocated!'). But I don't know details about how expensive operation the freeing of memory is (I know f.e. Vim doesn't allocate on heap in some time-critical functions to prevent leaks and expensive free()s), so I'm fine with removing free()s.